### ISSUE TITLE
What does security.mixed_content.upgrade_display_content do?

### ISSUE THREAD

**Issue by TearJoy** [arkenfox/user.js#1202]
So i was looking around about:config and I found the pref **security.mixed_content.upgrade_display_content**. What does it do? Based on the name, I assume it tries to upgrade insecure passive content to HTTPs, but what does it actually do? Is it fine to flip this pref?

Thanks in advance

**Comment by Thorin-Oakenpants** [comment]
- docs: [MDN](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content#upgrading_mixed-display_resources)
- tickets
   - [1633743](https://bugzilla.mozilla.org/show_bug.cgi?id=1633743) turned on in Nightly 84+ [patch](https://hg.mozilla.org/mozilla-central/rev/5d0b324f4397)
   - [1703847](https://bugzilla.mozilla.org/show_bug.cgi?id=1703847) turned off in Nightly 89+ (*and in general*) [patch](https://hg.mozilla.org/mozilla-central/rev/11262182f159)
   - [1672106](https://bugzilla.mozilla.org/show_bug.cgi?id=1672106) plans to turn on in release

Arkenfox enables HTTP-Only mode (see `1244`). I'm *think* HoM tries to upgrade everything: [this comment](https://bugzilla.mozilla.org/show_bug.cgi?id=1709838#c10) says (emphasis mine)
> If `HTTPS-Only` is enabled, **_all_** connections will be upgraded to `https`

And chris makes a point of difference with `HTTPS-First` not upgrading subresources, being top-level loads only

tl;dr: forget about it: when/if it is ready, firefox will enable it by default

**Comment by TearJoy** [comment]
Thanks as always!
